.github/workflows/pull_requests.yml

```diff
@@ -48,6 +48,14 @@ jobs:
           make helm-dependency-update
           make helm-lint
 
+  markdown-link-check:
+    runs-on: ubuntu-20.04
+    steps:
+    - uses: actions/checkout@master
```

Comment:
Is there a reason for `actions/checkout@master` and not `@v2` as the other actions?